experimental/coref/scripts/preprocess.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+# Convert conll annotations to DocBin
+# Coref uses a special CoNLL format, so the usual spaCy converters don't work.
+
+import sys
+import re
+from collections import defaultdict
+
+import spacy
+from spacy.tokens import Doc, DocBin
+
+DOCID_REGEX = "#begin document \((.*)\); part (\d*)"
+
+
+def read_file(fname, outname):
+    with open(fname, encoding="utf-8") as infile:
+        text = infile.read()
+
+    # TODO use larger model
```

Comment:
Is this TODO still relevant?

Reply:
No, it dates from when the small pipeline was used in development.

Reply:
I removed it.